code/modules/paperwork/paper_premade.dm

```diff
@@ -53,43 +53,6 @@
 	*	Stations
 	*/
 
-//////////	Cere fluff
```

Comment:
This sort of shit should be in the map files themselves imo. like map_files/Cerestation/fluff.dm